script/Makefile

```diff
@@ -97,6 +100,13 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+# Support sed on mac osx
```

Comment:
We discussed macOS support for the build scripts before and decided that macOS users use gnu-sed instead of the scripts take care of the compatibility. See #3398

Please rever the changes.

Reply:
Done